contrib/test.sh

```diff
@@ -10,15 +10,10 @@ fi
 
 pin_common_verions() {
```

Comment:
Nit: I think we can rename this into `cleanup`

Reply:
I was thinking of keeping it if some dependencies update their MSRV. But not really sure if it's better than renaming.